inventory-api/concepts/how-inventories-work.md

```diff
@@ -82,9 +82,22 @@ The request body contains the parameters for the inventory flow. In the case of
 ```
 See [Update inventory balance](../user-guides/manage-inventory-balances/update-inventory-balance.md). 
 
-### Event notifications
+### Inventory events
+
+Using the Zettle Pusher API, your application can get notified about changes to the inventory. You can create a webhook subscribing to inventory-related events. When the event occurs, a notification is sent to the specified notification URL. The notification is a `POST` request with  event details in JSON format.
+
 Inventory balance updates can be the result of either a purchase event, or an explicit update request. Such a request will trigger webhook event notifications for balance updates similar to purchase events.
 
+You can for example create a subscription for the `InventoryBalanceChanged` event. Whenever the inventory gets updated for example through a purchase, the `InventoryBalanceChanged` event occurs. Your application will then receive event data on the destination server. The event data is the payload for the updated inventory. 
```

Comment:
For example, your app subscribes to the `InventoryBalanceChanged` event. When a purchase happens, your app will receive data of the `InventoryBalanceChanged` event on the destination server, The event data is the payload for the updated inventory.

Reply:
Fixed